pyproject.toml

```diff
@@ -1,10 +1,25 @@
 [build-system]
 # Setuptools version should match setup.py; wheel because pip will insert it noisily
-requires = ["setuptools >= 38.3.0", "wheel"]
+requires = ["setuptools >= 42.0.0", "versioningit ~= 0.1.0", "wheel"]
 build-backend = 'setuptools.build_meta'
 
+[tool.versioningit]
+default-version = "0+unknown"
+
+[tool.versioningit.vcs]
+exclude = ["schema-*"]
+
+[tool.versioningit.format]
+# Same format as versioneer
+distance = "{version}+{distance}.{vcs}{rev}"
+dirty = "{version}+{distance}.{vcs}{rev}.dirty"
+distance-dirty = "{version}+{distance}.{vcs}{rev}.dirty"
```

Comment:
since it would be a common usecase AFAIK -- why `versiongit` doesn't just default to it (and also for above `default-version` which sounds like something to be within the same config section IMHO) and thus not require  all this boilerplate in every project?

Reply:
versioningit already has a default set of formats (listed at the end of [this section](https://github.com/jwodder/versioningit#the-tool-versioningit-format-subtable).  I just gave different ones here so that the versioning would continue to be consistent with versioneer.  As for `default-version`, that's for recovering from errors, which I believe the default response to should be the installer failing.